ps_qualityassurance.php

```diff
@@ -100,6 +103,12 @@ public function install()
      */
     public function __call($methodName, array $arguments)
     {
+        if (in_array($methodName, ['hookHeader', 'hookDisplayTop'])) {
+            // ps_qualityassurance hooks itself to listen to JS events
+            // so these hook calls must not be recorded
```

Comment:
If you prevent hooks from being callable, are they still working? You're using this hook on line #136, maybe you should use `actionFrontControllerSetMedia`?